spec/spec_helper.rb

```diff
@@ -4,6 +4,10 @@
 require 'rspec/rails'
 require 'capybara/email/rspec'
 require 'sidekiq/testing/inline'
+require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.